ui/app/helpers/utils/conversion-util.js

```diff
@@ -54,6 +54,11 @@ const baseChange = {
   BN: (n) => new BN(n.toString(16)),
 }
 
+// Utility function for checking base types
+const isValidBase = (base) => {
+  return base !== undefined
```

Comment:
Let's tighten up this type check:
```suggestion
  return base === 10 || base === 16
```

Reply:
The alternative would be this, but I don't know if we ever want anything other than 10 and 16. Use this if you're not sure:
```javascript
return Number.isInteger(base) && base > 1
```